models/dashboards.go

```diff
@@ -42,17 +51,18 @@ func buildIstioAggregations(local, remote string) []kmodel.Aggregation {
 	}
 	aggs = append(aggs, []kmodel.Aggregation{
 		{
-			Label:       fmt.Sprintf("%s_app", remote),
+			Label:       fmt.Sprintf("%s_%s", remote, appLabel),
 			DisplayName: "Remote app",
 		},
 		{
-			Label:       fmt.Sprintf("%s_version", remote),
+			Label:       fmt.Sprintf("%s_%s", remote, verLabel),
 			DisplayName: "Remote version",
 		},
 		{
 			Label:       "response_code",
 			DisplayName: "Response code",
 		},
+		// TODO: Do we need to aggrgate also on grpc_response_status?
```

Comment:
yes, we should add that

Reply:
OK, thanks.  I can open another issue for that, it's not related to this issue.

Reply:
https://github.com/kiali/kiali/issues/2485